### ISSUE TITLE
Shift key hangs other input

### ISSUE THREAD

**Issue by abstractMetaContext** [Aleph-One-Marathon/alephone#190]
I have been playing since the 90s with a button mapping that has secondary weapon fire on the left shift key. Often when pressing the shift key it makes the following movement key presses not resolve. This has been an issue since the first Aleph One release.

I revisit the games once a year or so and the annoyance of this issue is slowly eroding my sanity away.

Sincerely hope to see it fixed one day.

**Comment by treellama** [comment]
Curiously, I use left shift for action, and have never noticed this! Do you have any keyboard altering extensions installed such as AutoHotKey?

**Comment by abstractMetaContext** [comment]
Nope. also this is a longstanding issue for years. I use Windows if that is relevant.

**Comment by treellama** [comment]
Do you have sticky keys turned off? Sometimes pressing shift a bunch of times will activate it.

**Comment by abstractMetaContext** [comment]
Sticky keys is off.

**Comment by treellama** [comment]
Do you have access to another computer you can try this on? I'm not seeing any such problem here in Windows.

**Comment by abstractMetaContext** [comment]
It happens on my new build, and my own build and all other builds since i first installed Aleph One.

I use the numpad for movement, strafe running on 7 and 9. maybe this is relevant?
